src/deferred.js

```diff
@@ -0,0 +1,21 @@
+
+var oldDeferred = jQuery.Deferred;
+
+jQuery.Deferred = function() {
+	var deferred = oldDeferred.apply( this, arguments );
+
+	// Don't add this method if the current jQuery doesn't provide it
```

Comment:
I think you mean "only add this method if the current jQuery doesn't provide it"